control/trajectory_follower/src/longitudinal_controller_utils.cpp

```diff
@@ -62,17 +62,31 @@ bool isValidTrajectory(const Trajectory & traj)
   return true;
 }
 
-float64_t calcStopDistance(const Point & current_pos, const Trajectory & traj)
+float64_t calcStopDistance(
+  const Pose & current_pose, const Trajectory & traj, const float64_t & max_dist,
+  const float64_t & max_yaw)
```

Comment:
Same as above coment. & is not required for float64_t

Reply:
I fixed this commit 974004039d6a8aba053086a8b7cfff01da5294d7